src/affinder/__init__.py

```diff
@@ -2,3 +2,5 @@
     from ._version import version as __version__
 except ImportError:
     __version__ = "unknown"
+
+from .affinder import start_affinder
```

Comment:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

:stuck_out_tongue_winking_eye: